workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/components/RecentActivity.tsx

```diff
@@ -7,9 +7,9 @@ import RelativeTime from '@yaireo/relative-time';
 
 import {Ticket} from '../types';
 
-const relativeTime = new RelativeTime();
+const RELATIVE_TIME = new RelativeTime();
```

Comment:
This should still be camelCased, because it's not actually a static constant, it's a Class, so functions and classes shouldn't follow that type of format.